src/wasm-v8.cc

```diff
@@ -240,30 +217,22 @@ DEFINE_VEC(Val, vec, VAL)
 
 // Configuration
 
-struct ConfigImpl {
+struct ConfigImpl : public Config {
```

Comment:
Let's either convert all these definitions to `class` or drop the `public`.

Reply:
Dropping `public` breaks a lot, I've opted for making them classes.

Reply:
> Dropping public breaks a lot, I've opted for making them classes.

I'm confused, what can it break? Isn't public simply the default for structs?

Reply:
I'm not sure what I tested, maybe I tried `class ConfigImpl : Config` which defaults to private inheritance. `struct ConfigImpl : Config` works fine.